ko/news/_posts/2016-12-12-ruby-2-4-0-rc1-released.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: news_post
+title: "루비 2.4.0-rc1 릴리스"
+author: "naruse"
+translator: "yous"
+date: 2016-12-12 00:00:00 +0900
```

Comment:
This line or the file name should be updated: https://github.com/ruby/www.ruby-lang.org/pull/1525/files/c7bf27de906286557221b27243c0c6d7d45cb166#diff-5691c787a3d551f315888cca9cb0ad15